lib/src/model/robotoff_nutrient_extraction.dart

```diff
@@ -0,0 +1,201 @@
+import 'package:json_annotation/json_annotation.dart';
+import '../interface/json_object.dart';
+
+part 'robotoff_nutrient_extraction.g.dart';
+
+@JsonSerializable()
+class RobotoffNutrientExtractionResult extends JsonObject {
+  final String? status;
+  final int? count;
+  final List<NutrientExtractionInsight>? insights;
+
+  const RobotoffNutrientExtractionResult({
+    this.status,
+    this.count,
+    this.insights,
+  });
+
+  NutrientExtractionInsight? get getLatestInsights {
+    insights?.sort((a, b) => a.completedAt!.compareTo(b.completedAt!));
+    return insights?.last;
+  }
+
+  NutrientEntity? getNutrientEntity(String nutrientKey) {
+    return getLatestInsights?.data?.nutrients?[nutrientKey];
+  }
+
+  factory RobotoffNutrientExtractionResult.fromJson(
+          Map<String, dynamic> json) =>
+      _$RobotoffNutrientExtractionResultFromJson(json);
+
+  @override
+  Map<String, dynamic> toJson() =>
+      _$RobotoffNutrientExtractionResultToJson(this);
+}
+
+@JsonSerializable()
+class NutrientEntity {
+  final int? start;
+  final int? end;
+  final String? text;
+  final String? unit;
+  final double? score;
+  final bool? valid;
+  final String? value;
+  final String? entity;
+  @JsonKey(name: 'char_start')
+  final int? charStart;
+  @JsonKey(name: 'char_end')
+  final int? charEnd;
+
+  const NutrientEntity({
+    this.start,
+    this.end,
+    this.text,
+    this.unit,
+    this.score,
+    this.valid,
+    this.value,
+    this.entity,
+    this.charStart,
+    this.charEnd,
+  });
+
+  factory NutrientEntity.fromJson(Map<String, dynamic> json) =>
+      _$NutrientEntityFromJson(json);
+
+  Map<String, dynamic> toJson() => _$NutrientEntityToJson(this);
+}
+
+@JsonSerializable()
+class NutrientAnnotationData {
+  final String? unit;
+  final String? value;
```

Comment:
Quite obscure `String? unit` and `String? value`.
Are these randoms strings, or are we supposed to get typical values from a list.

Reply:
I've changed unit to be of type Unit, but value cannot be parsed as a number as it contains symbols like '<'

Reply:
> value cannot be parsed as a number as it contains symbols like '<'

Understood. A bit problematic as we don't support '<' in off-dart and Smoothie AFAIK. cf. https://github.com/openfoodfacts/openfoodfacts-dart/issues/584

Would be a good opportunity to
* create a `NutrientModifier` enum (<, >, ~)
* in your class, create 2 getters (`NutrientModifier? getNutrientModifier`, `double? getNutrientValue`)

Or something like that.

Reply:
Sounds good, I'm on it